app/src/main/java/de/danoeh/antennapod/adapter/AllEpisodesRecycleAdapter.java

```diff
@@ -118,8 +117,12 @@ public void onBindViewHolder(final Holder holder, int position) {
         holder.placeholder.setVisibility(View.VISIBLE);
         holder.placeholder.setText(item.getFeed().getTitle());
         holder.title.setText(item.getTitle());
+        int format = DateUtils.FORMAT_ABBREV_ALL;
```

Comment:
I thought about giving the feed item some sort of "getFormatedPubDate()" function, but it didn't seem right, somehow...